src/app/containers/Include/index.jsx

```diff
@@ -25,12 +28,30 @@ const componentsToRender = {
   },
 };
 
+const FallbackGrid = styled(GridItemConstrainedMedium)`
+  display: grid;
+`;
```

Comment:
I missed the psammead grid handover last week, in theory are these deprecated in favour of psammead grid? Just for my understanding really.

Reply:
I also missed the handover so not sure. I basically replicated what we have in the include containers which is using `styledGrid`.